packages/flex-plugin/src/utils/loadJS.ts

```diff
@@ -1,4 +1,4 @@
-import shortid from 'shortid';
+import shortid from './shortid';
```

Comment:
@ahcai this is the actual package that will be bundled with the customer's plugin. I removed the `shortid` dependency to make this lighter and replaced it with a one-liner file packages/flex-plugin/src/utils/shortid.ts